<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="generate">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>QRCode</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ean13country">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>123</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ean13ean">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>123456789</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Country</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>271</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>European Article Number 9 digits no spaces</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ean13">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>140</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EAN13</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eanproductname">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>291</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>productname</string>
    </property>
   </widget>
   <widget class="QPushButton" name="scan">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>250</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SCAN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="savetofile">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>88</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Savetofile</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="decode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>351</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
